lib/sandboxed_renderer/init.ts

```diff
@@ -80,7 +80,7 @@ webFrameInit();
 
 // Pass different process object to the preload script(which should not have
 // access to things like `process.electronBinding`).
-const preloadProcess = new EventEmitter();
+const preloadProcess: NodeJS.Process = new EventEmitter() as any;
```

Comment:
Does this work...?
```suggestion
const preloadProcess: NodeJS.Process & EventEmitter = new EventEmitter() as any;
```

Reply:
NodeJS.Process is an event emitter already